hasjob/views/helper.py

```diff
@@ -489,7 +545,7 @@ def list_dirty_impression_counts():
 
 def update_impression_counts(jobpost_ids):
     for jobpost_id in jobpost_ids:
-        # Replicate post.viewcounts's query here because we don't want to load the post from db just to
+        # Replicate viewcounts query from JobPost here because we don't want to load the post from db just to
```

Comment:
This excuse no longer applies as it's not in the JobPost class. The call could be abstracted into a separate function.